### DISCUSSION TITLE
Issue with 'dmarc.report' Table Not Being Created

### DISCUSSION THREAD

**Discussion by almogoren** [userjack6880/Open-DMARC-Analyzer#95] [Q&A] [answered]
Hello all,

I'm currently working on setting up the Open-DMARC-Analyzer and have run into a problem. Despite following the installation instructions, I'm encountering an error: "Open DMARC Analyzer failed: Table 'dmarc.report' doesn't exist".

I've verified that MySQL is running and the "dmarc" database is present, but it seems that the 'report' table is not being created within it. This leads to the error message when I try to use the DMARC Analyzer.

Any guidance on how to fix this would be greatly appreciated. Thanks in advance!

**Comment by userjack6880** [comment]
I suspect my emails have fallen into your junkmail folder (Google's bad at not trusting non-cloud providers). I'll copy it here.

This is a common setup error - you need to generate those tables using Open Report Parser as Open DMARC Analyzer uses those existing tables to create its view.

[Open Report Parser](https://github.com/userjack6880/Open-Report-Parser) is a requirement for ODA to work - it is what ingests data into the database.

**Comment by almogoren** [comment]
Dear John,

I appreciate your timely response. I have installed Open Report Parser, but
I am still facing some issues related to MySQL.

There's some confusion around the MySQL version support as stated in the
readme. The readme states that "Open DMARC Analyzer will run on any
webserver that supports PHP 7.4+ and MySQL 15.1+." As far as I'm aware,
there's no MySQL version 15.1, with the most recent release being 8.x.

I am currently using MySQL 8.0, and I would like to know if the analyzer
supports this version, or if I should downgrade to MySQL 5.7?

I'm looking forward to your guidance on this issue.

Kind regards,
Almog

**Reply by userjack6880** [reply]
You're right - I don't know where I got 15.1 from, and I'll need to update the documentation to reflect this. I appreciate pointing this out. Sorry for the long delay in response.

I targeted MariaDB 10.5, which is an LTS from 2019, and is based on MySQL 5.5 initially, and split around 2012. It's likely there's incompatibility between MariaDB 10 and MySQL 8.

**Comment by almogoren** [comment]
This is the error message I get when I try to install the install.php
If you can help me with this I would really appreciate any help as I have
been stuck for several days now and have not found a solution.

**Comment by almogoren** [comment]
Dear John,

I hope this email finds you well. apologize for the delay in my response
and express my appreciation for the updates you provided.

I'm glad to inform you that the issue has been resolved, and Open DMARC
Analyzer is now functioning properly with MariaDB.
Thank you for your assistance in identifying the compatibility requirements.

Best regards,
Almog

**Reply by userjack6880** [reply]
I apologize for not getting back to you quickly after my last response - how did you resolve your issue? It would be helpful for anyone else that comes across the same issue.

EDIT: I need to drink coffee before responding to these - I see you switched to MariaDB from MySQL.

**Comment by almogoren** [comment, marked as answer]
No problem at all regarding the delayed response. To address the issue, I
simply reinstalled the entire system using MariaDB, and that solved the
compatibility problems. Open DMARC Analyzer is now working smoothly.
